scripts/kind-e2e/e2e.sh

```diff
@@ -4,11 +4,13 @@
 
 source /usr/share/shflags/shflags
 DEFINE_string 'deploytool' 'operator' 'Tool to use for deploying (operator/helm)'
+DEFINE_string 'focus' 'all' 'Ginkgo focus for the E2E tests'
```

Comment:
Since focus is a regex you can set the default to '.*' and it should work always if you pass $focus as is to -ginkgo.focus